accounts/templates/accounts/login.html

```diff
@@ -44,12 +44,14 @@ <h3 class="heading">{% trans "Login" %}</h3>
             <em class="fa fa-sign-in"></em>
             {% trans "Log in" %}
           </button>
+          <!--
```

Comment:
Maybe will be better if we use the [comment template tag](https://docs.djangoproject.com/en/3.1/ref/templates/builtins/#comment)

`{% comment %}...{% endcomment %}`

Reply:
done